my_list.rb

```diff
@@ -0,0 +1,28 @@
+require_relative './my_enumerable'
+
+# My List
+class MyList
+  # Loads my enumerable methods
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    return to_enum(:each) unless block_given?
+
+    @list.each(&block)
```

Comment:
- [ ] [OPTIONAL] Even though this works, it can still be improved with a `for` or `while` loop to create a custom `each` method.

Reply:
Ok. Thanks I didnt see that. Noted